arkane/explorer.py

```diff
@@ -250,9 +250,9 @@ def execute(self, output_file, plot, file_format='pdf', print_summary=True, spec
             for rxn in network.path_reactions:
                 if rxn.transition_state is None:
                     if rxn.network_kinetics is None:
-                        E0 = sum([spec.conformer.E0.value_si for spec in rxn.reactants]) + rxn.kinetics.Ea.value_si
+                        E0 = sum([spec.conformer.E0.value_si for spec in rxn.reactants]) + rxn.kinetics.Ea.value_si + network.energy_correction
                     else:
-                        E0 = sum([spec.conformer.E0.value_si for spec in rxn.reactants]) + rxn.network_kinetics.Ea.value_si
+                        E0 = sum([spec.conformer.E0.value_si for spec in rxn.reactants]) + rxn.network_kinetics.Ea.value_si + network.energy_correction
```

Comment:
Could you explain? What is this correction?

Reply:
So as I was digging through things I noticed that the E0's for everything don't line up between networks. This occurs because the networks each choose their own correction to the energies that seem to be intended to keep the values closer to zero: https://github.com/ReactionMechanismGenerator/RMG-Py/blob/2cdc2ea7cd53e6a09648c6c50bc936812207fc81/rmgpy/rmg/pdep.py#L822. This correction is included when this same operation is done during network updates: https://github.com/ReactionMechanismGenerator/RMG-Py/blob/2cdc2ea7cd53e6a09648c6c50bc936812207fc81/rmgpy/rmg/pdep.py#L857.